event.coffee

```diff
@@ -11,7 +11,7 @@ axios = require 'axios'
   compact
 } = require 'underscore'
 
-URL_REGEX = /\b(https?:\/\/\S+)\b/
+URL_REGEX = /(https?:\/\/[0-9a-zA-Z\+\&\?\:\=\.\/\-]+)/
```

Comment:
There are probably better regular expressions we can use here like the ones from this SO thread (https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url), but we chose to use a simpler version. Let me know what you think!

Reply:
✅ tested the weird urlencoded version that gcal link sends us as well:
![image](https://github.com/kieran/torontojs-event-bot/assets/3444/b7ca6858-ff2c-4b06-b430-f1805712f96f)